<launch>
    <node pkg="bebop_controller" type="bebop_joy.py" name="bebop_joy" output="screen">
        <rosparam command="load" file="$(find bebop_controller)/config/controller_gains.yaml" />
    </node>
    <node pkg="joy" type="joy_node" name="joystic" />
    <include file="$(find vicon_bridge)/launch/vicon_lab.launch" />
    <include file="$(find bebop_tools)/launch/bebop_nodelet_iv.launch" />


    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find bebop_controller)/config/BebopConfig.rviz" />
</launch>
